architecture/additional_concepts/authentication.adoc

```diff
@@ -180,15 +180,15 @@ $ oc create -f <(echo '
 
 *Service Accounts as OAuth Clients*
 
-A service account can be used as a very constrained kind of OAuth client.
-Service accounts can only request a subset of scopes that allow
-access to some basic user information and power inside of the service account's own namespace.
-Specifically,  `user:info`, `user:check-access`, `role:<anyrole>:<serviceaccount.namespace only>[:!]`.
-
-* `client_id` is `system:serviceaccount:<namespace>:<serviceaccount.name>`
-* `client_secret` can be any of the API tokens for that service account (`oc sa get-token serviceaccount.name`)
-* `redirect_uris` come from any annotation on the service account that has the prefix `serviceaccounts.openshift.io/oauth-redirecturi.`.
-  For example: `serviceaccounts.openshift.io/oauth-redirecturi.one`.
+A link:../../admin_guide/service_accounts.html[service account] can be used as a
+constrained form of OAuth client. Service accounts can only request a subset of
+scopes that allow access to some basic user information and power inside of the
+service account's own namespace. Specifically, `user:info`, `user:check-access`,
+and `role:<anyrole>:<serviceaccount.namespace only>[:!]`.
+
+* `client_id` can be used for `system:serviceaccount:<namespace>:<serviceaccount.name>`.
```

Comment:
A `client_id` is an identifier that is used to identify the client to the oauth server.  In this case, the value of the identifier is `system:serviceaccount:<namespace>:<serviceaccount.name>`.